apps/api/src/app/workflows-v1/notification-template.controller.ts

```diff
@@ -107,8 +113,10 @@ export class NotificationTemplateController {
         tags: body.tags,
         description: body.description,
         workflowId: body.identifier,
-        critical: body.critical,
-        preferenceSettings: body.preferenceSettings,
+        defaultPreferences: DEFAULT_WORKFLOW_PREFERENCES,
```

Comment:
All V1 Workflows now write a `defaultPreferences` to provide a default for the preference merging behaviour.